README.md

```diff
@@ -1,43 +1,82 @@
-**Addons** is a repository of contributions that conform to
+# TensorFlow Addons
+
+TensorFlow Addons is a repository of contributions that conform to
 well-established API patterns, but implement new functionality
 not available in core TensorFlow. TensorFlow natively supports
-a larger number of operators, layers, metrics, losses, and optimizers.
+a large number of operators, layers, metrics, losses, and optimizers.
 However, in a fast moving field like ML, there are many interesting new
 developments that cannot be integrated into core TensorFlow
-(because their broad applicability is not yet clear, or it is mostly used by a smaller
-subset of the community).
+(because their broad applicability is not yet clear, or it is mostly
+ used by a smaller subset of the community).
 
-# Scope
-The tensorflow/addons repository, will contain additional functionality fitting the following criteria:
+## Contents
+| Sub-Package    | Addon  | Reference                                  |
```

Comment:
The table seems redundant, and also might become a too long list to maintain in the future.

Reply:
I agree this is not sustainable in the long term. I would like a single list of whats available so new users can quickly see if what they need is available, without having to dig through the repo. Are you okay with keeping this until we have setup our Python API docs to show package contents.

For example: https://pytorch.org/docs/master/torchvision/

Reply:
Sorry for the delay. Great, that's what I think. We can remove it after API docs is set up in the future.